### ISSUE TITLE
can DTR support YOLOv5?

### ISSUE THREAD

**Issue by chaoyanghe** [uwsampl/dtr-prototype#11]
I want to try this method in memory contrained edge devices for YOLOv5 (https://github.com/ultralytics/yolov5), could you please help to have an example for it? thanks.

**Comment by MarisaKirisame** [comment]
Yes it can. I will write a more in-detail dtr-usage doc later (is on a deadline this week), but basically you have to build dtr yourself, then call .checkpoint() for the input tensor, and .apply_.(lambda x: x.detach().checkpoint()) for the model. then you are good
